src/style.css

```diff
@@ -1 +1,66 @@
 /* Styles go here */
+body {
```

Comment:
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md
```suggestion
.body {
```

Reply:
In each work you have `Checklist` and each time it is different
in this task you have hint №1 from `Checklist` which said next:

<img width="993" alt="image" src="https://user-images.githubusercontent.com/99652389/179195501-a49846f6-6ae1-4f6b-96e3-d19afb9e8321.png">

Reply:
Thanks for review.Changes was done, check please last commit.😄